src/test/java/io/reactiverse/pgclient/RowTest.java

```diff
@@ -97,6 +97,44 @@ public void testGetNonExistingRows(TestContext ctx) {
     }));
   }
 
+  @Test
+  public void testGetColumnNameRows(TestContext ctx) {
+    Async async = ctx.async();
+    PgClient.connect(vertx, options, ctx.asyncAssertSuccess(conn -> {
+      conn.query("SELECT 2 \"foo\"",
+        ctx.asyncAssertSuccess(result -> {
+          Row row = result.iterator().next();
+          ctx.assertEquals("foo",row.getColumnName(0));
+          async.complete();
+        }));
+    }));
+  }
+
+  @Test
+  public void testNegativeNotEqualGetColumnNameRows(TestContext ctx) {
+    Async async = ctx.async();
+    PgClient.connect(vertx, options, ctx.asyncAssertSuccess(conn -> {
+      conn.query("SELECT 2 \"foo\"",
+        ctx.asyncAssertSuccess(result -> {
+          Row row = result.iterator().next();
+          ctx.assertNotEquals("bar",row.getColumnName(0));
+          async.complete();
+        }));
+    }));
+  }
+
+  @Test
+  public void testNegativeGetColumnNameRows(TestContext ctx) {
+    Async async = ctx.async();
+    PgClient.connect(vertx, options, ctx.asyncAssertSuccess(conn -> {
+      conn.query("SELECT 2 \"foo\"",
+        ctx.asyncAssertSuccess(result -> {
+          Row row = result.iterator().next();
+          ctx.assertEquals(null,row.getColumnName(1));
```

Comment:
Should use `ctx.assertNull(row.getColumnName(-1))` instead

Reply:
@EmadAlblueshi this is still a valid test case (edge length test case), maybe we need to add an additional test case.

Reply:
@pratikpparikh Yes because this test should reflects its name :)

Reply:
@EmadAlblueshi agreed.  The test cases are now named appropriately.

Reply:
@EmadAlblueshi we should be good with test case naming now as well.